trl/trainer/ppo_trainer.py

```diff
@@ -461,15 +461,21 @@ def step(
                 model_inputs["decoder_attention_mask"] = self.accelerator.pad_across_processes(
                     model_inputs["decoder_attention_mask"], dim=1, pad_index=0, pad_first=pad_first
                 )
+            else:
+                model_inputs['labels'] = self.accelerator.pad_across_processes(
```

Comment:
We actually don't need labels, right? In `prepare_model_inputs` we pop them for encoder-decoder, I think we should do the same for encoders there.